server/api/attendance/attendance.controller.js

```diff
@@ -0,0 +1,674 @@
+'use strict';
+
+var _ = require('lodash');
+var Attendance = require('./attendance.model');
+var mongoose = require('mongoose');
+var ClassYear = require('../classyear/classyear.model');
+var User = require('../user/user.model');
+var SmallGroup = require('../smallgroup/smallgroup.model');
+var config = require('../../config/environment');
+
+function isoDateToTime(isoDate){
+    var date = new Date(isoDate);
+    date.setHours(0,0,0,0);
+    return date.getTime();
+}
+
+// *******************************************************
+// Get list of attendance submissions
+// Restricted to mentors
+// router.get('/', auth.isAuthenticated(), controller.index);
```

Comment:
It is much simpler to look at this way, than having to cross reference 2 files with the front end code. It also allows better searchability and removes the need to look at the index.html most of the time. You should be looking at the usage of the endpoint anyway if you are changing the permissions (and that does not ofter happen)